.kokoro/presubmit/downstream-build.sh

```diff
@@ -39,8 +39,8 @@ GAPIC_BOM_VERSION=$(sed -e 's/xmlns=".*"//' gapic-generator-java-bom/pom.xml | x
 
 ### Round 2
 # Run the updated GAPIC BOM against HEAD of java-shared-dependencies
-git clone "https://github.com/googleapis/java-shared-dependencies.git" --depth=1
-pushd java-shared-dependencies/first-party-dependencies
+git clone "https://github.com/googleapis/google-cloud-java.git" --depth=1
+pushd google-cloud-java/java-shared-dependencies/first-party-dependencies
```

Comment:
It's working. [The log](https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgapic-generator-java%2Fpresubmit%2Fgraalvm-native-17-downstream/activity/455be2c9-382a-4517-8d28-b7482be72f37/log) shows;

```
diff --git a/java-shared-dependencies/first-party-dependencies/pom.xml b/java-shared-dependencies/first-party-dependencies/pom.xml
index 0bf109d..6598701 100644
--- a/java-shared-dependencies/first-party-dependencies/pom.xml
+++ b/java-shared-dependencies/first-party-dependencies/pom.xml
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>first-party-dependencies</artifactId>
@@ -36,7 +34,7 @@
         <!-- This BOM declares the versions gRPC, Protobuf, Guava, etc. -->
         <groupId>com.google.api</groupId>
         <artifactId>gapic-generator-java-bom</artifactId>
-        <version>${gapic-generator-java-bom.version}</version>
+        <version>2.14.1-SNAPSHOT</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
```